src/environments/environment.e2e.ts

```diff
@@ -0,0 +1,18 @@
+import { Environment } from 'src/app/shared/helpers/config';
+
+export const environment: Environment = {
+  production: false,
+  apiUrl: 'https://dev.algorea.org/api',
+  oauthServerUrl: 'https://login.france-ioi.org',
+  oauthClientId: '43',
+
+  defaultActivityId: '4702',
+  defaultSkillId: '3000',
+
+  languages: [
+    { tag: 'fr', path: '/fr/' },
+    { tag: 'en', path: '/' },
+  ],
+  allowForcedToken: true,
+  authType: 'tokens',
+};
```

Comment:
For e2e testing in CI, I had 2 options:
1. Launch the mock server in CI and test over it. For information, the mock server currently calls dev algorea api behind the scene when no mock is available for a route.
2. Create another env in which we can select the api url, and fill it with dev algorea api url instead of mock server for now.

I took the initiative to pick option 2. and create+use a dedicated `e2e` environment. This stay we stay iso with the current CI.
I think when the mock server will be more populated with data, then it will be interesting to operate a switch and use the mock server to create specific e2e scenarios.